pencilcode/files/systemd/pencilcode-server.service

```diff
@@ -0,0 +1,14 @@
+[Service]
+WorkingDirectory = /home/codio/pencilcode/
+User = codio
+Group = codio
+TimeoutStopSec = 10
+ExecStart = /home/codio/pencilcode/pencilcode-server.sh
```

Comment:
I think we can run just `/usr/bin/npm run serve` and remove pencilcode-server.sh